ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/ParallelIndexerSubscription.scala

```diff
@@ -180,6 +178,7 @@ object ParallelIndexerSubscription {
       previous: Batch[Vector[DbDto]],
       current: Batch[Vector[DbDto]],
   ): Batch[Vector[DbDto]] = {
```

Comment:
Since here we want to time one contiguous synchronous execution path, can you use `Timed.value(..)`? (e.g. usage in [here](https://github.com/digital-asset/daml/blob/main/ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/bridge/validate/SequenceImpl.scala#L60-L61))